src/mono/mono/metadata/assembly.c

```diff
@@ -1478,45 +1475,27 @@ bundled_assembly_match (const char *bundled_name, const char *name)
 }
 
 static MonoImage *
-open_from_bundle_internal (MonoAssemblyLoadContext *alc, const char *filename, MonoImageOpenStatus *status, gboolean is_satellite)
+open_from_bundle_internal (MonoAssemblyLoadContext *alc, const char *filename, MonoImageOpenStatus *status)
 {
-	if (!bundles)
```

Comment:
I guess we should still handle the fast path by using !mono_bundled_resources_contains_assemblies ()